docs/sources/enterprise/access-control/permissions.md

```diff
@@ -23,50 +23,50 @@ scope
+| `users.quotas:update`      | `global:users:*`                                                                        | Update a user’s quotas.                                                                                                                                  |
+| `org.users.read`           | `users:*`                                                                               | Get user profiles within an organization.                                                                                                                |
+| `org.users.add`            | `users:*`                                                                               | Add a user to an organization.                                                                                                                           |
+| `org.users.remove`         | `users:*`                                                                               | Remove a user from an organization.                                                                                                                      |
+| `org.users.role:update`    | `users:*`                                                                               | Update the organization role (`Viewer`, `Editor`, `Admin`) for an organization.                                                                          |
+| `ldap.user:read`           | n/a                                                                                     | Get a user via LDAP.                                                                                                                                     |
+| `ldap.user:sync`           | n/a                                                                                     | Sync a user via LDAP.                                                                                                                                    |
+| `ldap.status:read`         | n/a                                                                                     | Verify the LDAP servers’ availability.                                                                                                                   |
+| `ldap.config:reload`       | n/a                                                                                     | Reload the LDAP configuration.                                                                                                                           |
+| `status:accesscontrol`     | `services:accesscontrol`                                                                | Get access-control enabled status.                                                                                                                       |
```

Comment:
What is happening here? This sounds a bit cryptic. What is the user’s goal here?

Reply:
The goal is to know whether `fine grained access control` is in use. If the status says `disabled`, then, we are not enforcing fined-grained access control, but rather the legacy access control based on the organization role solely.